src/corehost/cli/nethost/nethost.h

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef __NETHOST_H__
+#define __NETHOST_H__
+
+#if defined(_WIN32)
+    #ifdef NETHOST_EXPORT
+        #define NETHOST_API __declspec(dllexport)
+    #else
+        #define NETHOST_API __declspec(dllimport)
+    #endif
+
+    #define NETHOST_CALLTYPE __stdcall
```

Comment:
@vitek-karas Bah... Yes, you are right. The rest of the core-setup API is `__cdecl`. I agree that using `__stdcall` is more consistent with `coreclr.dll`, `comhost`, `ijwhost`, and the proposed `winrthost` exports. I will say this is annoying since all of `hostpolicy` and `hostfxr` are `__cdecl`. So much annoyance here. Thanks for pointing this out. I think we should keep with `__stdcall` since it aligns with all non "internal" core-setup scenarios better. I consider interacting directly with `hostfxr` and `hostpolicy` more "internal" or at the very least "advanced" as compared to COM, IJW, WinRT, and the native loading scenarios which users could experiment with.

Reply:
Agreed - and I like the consistency among the "public" hosts. `hostpolicy` is not going to be used externally (it's just too hard to use directly). `hostfxr` uses will be advanced and so some complexity is OK.

Reply:
@elinor-fung In a previous review @vitek-karas mentioned the need for documentation of these kinds of contracts. I think the fact that we are starting to coalesce around common user scenarios using `__stdcall` on x86 is an important contract that should be documented and officially defined. I have observed you reading through the documentation of core-setup, is there an existing doc that could contain these kind of details or is a new one needed?

Reply:
I'd say we need a new one. Maybe what gets created for #5481 would be a reasonable place (if it is a doc for all exposed APIs - I could also see doing a separate 'advanced' `hostfxr`/`hostpolicy` doc and 'public' `*host` doc)